include/ppx/metrics.h

```diff
@@ -141,7 +123,8 @@ class MetricGauge final
     };
 
 private:
-    ~MetricGauge();
+    MetricGauge(const MetricMetadata& metadata)
+        : mMetadata(metadata), mAccumulatedValue(0) {}
```

Comment:
isn't the mAccumulatedValue redundant with the other default value below?

Reply:
yes! good catch, thanks.